x-pack/legacy/plugins/snapshot_restore/public/app/sections/home/policy_list/policy_retention_schedule/policy_retention_schedule.tsx

```diff
@@ -0,0 +1,212 @@
+        <EuiSpacer />
+      </Fragment>
+    );
+  }
+
+  if (retentionSettings && retentionSettings.retentionSchedule) {
+    const { retentionSchedule } = retentionSettings;
+    return (
+      <Fragment>
+        <EuiPanel>
+          <EuiFlexGroup justifyContent="spaceBetween" alignItems="center">
+            <EuiFlexItem grow={false}>
+              <EuiText>
+                <p>
+                  {retentionSchedule ? (
+                    <FormattedMessage
+                      id="xpack.snapshotRestore.policyRetentionSchedulePanel.retentionScheduleText"
+                      defaultMessage="Snapshot retention will run on the cron schedule: {cronSchedule}."
+                      values={{ cronSchedule: <strong>{retentionSchedule}</strong> }}
+                    />
+                  ) : (
+                    <FormattedMessage
+                      id="xpack.snapshotRestore.policyRetentionSchedulePanel.retentionScheduleNotConfiguredText"
+                      defaultMessage="A snapshot retention schedule is not configured."
+                    />
+                  )}
+                </p>
+              </EuiText>
+            </EuiFlexItem>
+            <EuiFlexItem grow={false}>
+              {retentionSchedule ? (
+                <PolicyUpdateRetentionProvider>
+                  {updateRetentionPrompt => {
+                    return (
+                      <EuiToolTip
+                        position="top"
+                        content={
+                          <FormattedMessage
+                            id="xpack.snapshotRestore.policyRetentionSchedulePanel.retentionScheduleEditLinkTooltipText"
+                            defaultMessage="Edit retention schedule"
+                          />
+                        }
+                      >
+                        <EuiButtonIcon
+                          iconType="pencil"
+                          onClick={() =>
+                            updateRetentionPrompt(retentionSchedule, onRetentionScheduleUpdated)
+                          }
+                          aria-label={i18n.translate(
+                            'xpack.snapshotRestore.policyRetentionSchedulePanel.retentionScheduleEditLinkAriaLabel',
+                            {
+                              defaultMessage: 'Edit retention schedule',
+                            }
+                          )}
+                        />
+                      </EuiToolTip>
+                    );
+                  }}
+                </PolicyUpdateRetentionProvider>
+              ) : (
+                <PolicyUpdateRetentionProvider>
+                  {updateRetentionPrompt => {
+                    return (
+                      <EuiToolTip
+                        position="top"
+                        content={
+                          <FormattedMessage
+                            id="xpack.snapshotRestore.policyRetentionSchedulePanel.retentionScheduleAddLinkTooltipText"
```

Comment:
I believe there is no need for the `Text` keyword. Applies everywhere else as well.
The ids should end with the following:  https://github.com/elastic/kibana/blob/master/packages/kbn-i18n/GUIDELINE.md#message-types